apis/Google.Cloud.Logging.Log4Net/Google.Cloud.Logging.Log4Net/GoogleStackdriverAppender.cs

```diff
@@ -194,6 +201,38 @@ public override void ActivateOptions()
             _isActivated = true;
         }
 
+        private LoggingServiceV2Client BuildLoggingServiceClient()
+        {
+            GoogleCredential credential;
+            if(!TryGetCredentialFromFile(out credential))
+            {
+                return LoggingServiceV2Client.Create();
+            }
+            
+            return LoggingServiceV2Client.Create(new Grpc.Core.Channel(LoggingServiceV2Client.DefaultEndpoint.Host, LoggingServiceV2Client.DefaultEndpoint.Port, Grpc.Auth.GoogleGrpcCredentials.ToChannelCredentials(credential)));

```

Comment:
Please make the `.ToChannelCredentials()` call an extension method cal on `credential`. This will also remove the fully qualified `ToChannelCredentials` name here which will be better.

Please don't use the fully qualified `Grpc.Core.Channel` name; please have a suitable `using` statement.

And split this line if it's too long; ~120 characters, we don't have a hard limit, but it looks longer than all the lines around it.